conan/cli/formatters/graph/graph.py

```diff
@@ -99,6 +100,14 @@ def _render_graph(graph, template, template_folder):
 def format_graph_html(result):
     graph = result["graph"]
     conan_api = result["conan_api"]
+    field_filter = result["field_filter"]
+    if field_filter and "requires" not in field_filter:
```

Comment:
should it check for `field_filter is not None` instead (as empty list may evaluate to False in python)?

Reply:
Removing this, it doesn't really apply